actors/builtin/verifreg/verified_registry_test.go

```diff
@@ -749,6 +749,114 @@ func TestRestoreBytes(t *testing.T) {
 	})
 }
 
+//func TestRemoveVerifiedClientDataCap(t *testing.T) {
```

Comment:
are these tests ready to go? why commented out?

Reply:
nope - we will add more later on

Reply:
removing